iceoryx2-bb/cxx/tests/src/duration_tests.cpp

```diff
@@ -12,24 +12,20 @@
 //
```

Comment:
Our usual naming convention for test names seems to be to use snake_case for the test names and CamelCase for the test case name, while this file (and the chrono_support_tests below) uses CamelCase throughout for both.

Reply:
I meant that I would like to replace the legacy `string`, `optional` and `expected` in the public API with the their iceoryx2 counterparts before the release. Putting effort into renaming all the test names does not help with replacing the other legacy components and there is not that much time until the release. Since the test names doe not have any effect, except consistency, I would like to postpone this work until the important, public facing changes are finished. This means either leaving the test file where it is right now or moving the test file back to the legacy folder.